fastapi_sqla/__init__.py

```diff
@@ -139,15 +140,12 @@ class Meta(BaseModel):
     page_number: int = Field(..., description="Current page number. Starts at 1.")
 
 
-class Paginated(Collection, Generic[T]):
-    """Paginated collection with information on current page and total items in meta."""
+class Page(Collection, Generic[T]):
```

Comment:
if you want the generic to be applied to the subclass, don't you have to pass `[T]` to it? 
```suggestion
class Page(Collection[T], Generic[T]):
```

Reply:
Failed tests when doing so:
> `TypeError: Cannot parameterize a concrete instantiation of a generic model`